filebeat/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# 0.1.1
+[giladsh]
+- fixed registrty file parsing error - the file contains a list of dictionaries, not a dictionary
+- added and explicit file permission issue - since filebeat creates the file as root
```

Comment:
That's very unfortunate... and I can't seem to find anything in their doc as to how to change that :-/ any suggestions as to how to best do this in an automated way? In particular, does filebeat ever re-create the file after creating the first time?

Reply:
(just checked, it seems it does re-create it periodically, so just changing the permissions in Chef or what not won't be enough, damn... how did you go around that?)